t/thrift/thrift_ubi_8.5.sh

```diff
@@ -0,0 +1,94 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: thrift
+# Version	: v0.13.0
+# Source repo	: https://github.com/apache/thrift
+# Tested on	: UBI 8.5
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Atharv Phadnis <Atharv.Phadnis@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=thrift
+PACKAGE_VERSION=${1:-v0.13.0}
+PACKAGE_URL=https://github.com/apache/thrift
+
+# Include CentOS Repos
+dnf -y install \
```

Comment:
@atharv-phadnis is there a way to avoid using the centos repositories? which package (s) are being installed from there? We are not sure if every one would be open to enabling this on RHEL/UBI, also can you check if these are available on RHEL?

Reply:
@seth-priya It was required for installation of group "Development Tools", which wasn't available in the UBI repos. Main requirement from the groups is the **bison** package, I've tried building it from source, it gets build and installed successfully, but results in error during the building of **thrift** itself.

Reply:
Updated the script as per your suggestions, please review.